lib/msf/core/exploit/smb/client/pipe_auditor.rb

```diff
@@ -40,6 +40,9 @@ def check_named_pipes(check_first: [], return_first: false)
     named_pipes.each do |pipe|
       begin
         pipe_name   = pipe.strip
+        unless pipe_name.start_with?("\\")
+            pipe_name = "\\#{pipe_name}"
```

Comment:
Two-space indents, please.

Reply:
And please put blank lines around this conditional for readability.

Reply:
Fixed